### PR TITLE
Added K6 test for maskinportenschema with offeredby's org number in header

### PR DESCRIPTION
**Pull request by ivartryti**
## Description
Added a test that delegates maskinportenschema using offeredby's orgnumber in header.

## Related Issue(s)
- #277 

## Developer/Reviewer Checklist
- [ ] **Your** code builds clean without any errors or warnings
- [ ] No changes to config/appsettings or environment variables created in separate linked PR
- [ ] Manual testing done (required)
- [ ] Relevant integration test added (required for functional changes)
- [ ] Relevant automated test added (required for functional changes)
- [ ] All integration tests run green

## Documentation
- [ ] User documentation is updated with a separate linked PR in [altinn-studio-docs.](https://github.com/Altinn/altinn-studio-docs) (if applicable)

### CODE REVIEW

File: test/K6/src/api/platform/authorization/maskinporten.js [hunk 1]
```diff
@@ -78,6 +78,25 @@ export function postMaskinportenSchema(altinnToken, offeredByPartyId, resourceid
   return http.post(endpoint, bodystring, params);
 }
 
+/**
+ * GET call to get maskinportenschemas that have been received by the current party
+ * @param {*} altinnToken personal token for DAGL
+ * @param {*} offeredByOrganizationNumber the organization number for the offeredby party
+ * @param {*} resourceid the id of the resource to delegate
+ * @param {*} attributeId the attribute id for the receiver of the schema. 'urn:altinn:partyid' or 'urn:altinn:organizationnumber'
```
Comment (jonkjetiloye) [inline test/K6/src/api/platform/authorization/maskinporten.js:86]: attributeId and attributeValue could be better named to represent the party the delegation is made to

File: test/K6/src/api/platform/authorization/maskinporten.js [hunk 2]
```diff
@@ -78,6 +78,25 @@ export function postMaskinportenSchema(altinnToken, offeredByPartyId, resourceid
...
+ * @param {*} attributeValue the receiver's partyid or organization number
+ */
+export function postMaskinportenSchemaOrgNoInHeader(altinnToken, offeredByOrganizationNumber, resourceid, attributeId, attributeValue) {
+  var endpoint = config.buildMaskinPorteSchemaUrls('organization', 'maskinportenschema');
+  console.log(endpoint)
+  var params = header.buildHeaderWithRuntimeOrgNumberAndJson(altinnToken, 'personal', offeredByOrganizationNumber);
```
Comment (jonkjetiloye) [inline test/K6/src/api/platform/authorization/maskinporten.js:92]: what's the "personal" parameter?

File: test/K6/src/api/platform/authorization/maskinporten.js [hunk 3]
```diff
@@ -78,6 +78,25 @@ export function postMaskinportenSchema(altinnToken, offeredByPartyId, resourceid
...
+ * GET call to get maskinportenschemas that have been received by the current party
+ * @param {*} altinnToken personal token for DAGL
+ * @param {*} offeredByOrganizationNumber the organization number for the offeredby party
+ * @param {*} resourceid the id of the resource to delegate
+ * @param {*} attributeId the attribute id for the receiver of the schema. 'urn:altinn:partyid' or 'urn:altinn:organizationnumber'
+ * @param {*} attributeValue the receiver's partyid or organization number
+ */
+export function postMaskinportenSchemaOrgNoInHeader(altinnToken, offeredByOrganizationNumber, resourceid, attributeId, attributeValue) {
+  var endpoint = config.buildMaskinPorteSchemaUrls('organization', 'maskinportenschema');
+  console.log(endpoint)
```
Comment (jonkjetiloye) [inline test/K6/src/api/platform/authorization/maskinporten.js:91]: console.log for debugging purposes I presume? Is it only for local debugging or for pipeline execution?

Maybe we should have a debug helper of some sort. Where a debug context dictates whether the logging is performed or not?

File: test/K6/src/api/platform/authorization/maskinporten.js (hunk 2 above)
Comment (ivartryti) [reply test/K6/src/api/platform/authorization/maskinporten.js]: I saw that the 'personal' parameter was used to choose an endpoint in token-generator.js, and didn't dig any deeper when I saw it here. I see now that it's unused, so I removed it.

File: test/K6/src/api/platform/authorization/maskinporten.js [hunk 4]
```diff
@@ -78,6 +78,25 @@ export function postMaskinportenSchema(altinnToken, offeredByPartyId, resourceid
   return http.post(endpoint, bodystring, params);
 }
 
+/**
+ * GET call to get maskinportenschemas that have been received by the current party
```
[the remark this answers is not in the document]
Comment (ivartryti) [reply test/K6/src/api/platform/authorization/maskinporten.js]: Fixed

File: test/K6/src/api/platform/authorization/maskinporten.js (hunk 1 above)
Comment (ivartryti) [reply test/K6/src/api/platform/authorization/maskinporten.js]: renamed them to toAttributeId/toAttributeValue fromAttributeId/FromAttributeValue

**Review by jonkjetiloye — APPROVED** [review]

**Pull request merged**